examples/odata-cli/src/activities.ts

```diff
@@ -45,6 +46,7 @@ export async function useCatalogAndFetchSomeMetadata(
             });
         }
     } catch (error) {
+        // eslint-disable-next-line no-console
         console.error(error.cause || error.toString() || error);
```

Comment:
Should be using the logging module?

Reply:
This is an example. So console should be ok?

Reply:
Agree with @devinea. Example code, let's keep it simple and focussed